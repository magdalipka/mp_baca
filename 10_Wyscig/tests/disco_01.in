2
2 5
aa 1 2 3 4 5
bb 100 200 300 400 500
2 5
aa 1 2 3 4 5
bb 1 2 3 4 5
