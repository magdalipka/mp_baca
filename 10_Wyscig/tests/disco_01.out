aa - 0 pkt.
bb - 5 pkt.
aa - 0 pkt.
bb - 0 pkt.
